src/functionloading.jl

```diff
@@ -16,9 +16,9 @@ macro glfunc(opengl_func)
     func_name_str   = string(func_name)
     ptr_expr        = :(getprocaddress_e($func_name_str))
 
-    @windows_only begin # windows has some function pointers statically available and some not, this is how we deal with it:
+    if is_windows() # windows has some function pointers statically available and some not, this is how we deal with it:
```

Comment:
it's not necessary ;) Since this is a macro, this will be figured out at compile time anyways. I should also remove the `@static` in the top level definitions.